plugins/intel-me/README.md

```diff
@@ -0,0 +1,19 @@
+# Intel ME
+
+## Introduction
+
+This plugin is used to talk to the Intel ME device, typically CSME.
+
+This allows us to get the public key for BootGuard.
+
+## GUID Generation
+
+These devices use the existing GUIDs provided by the ME host interfaces.
+
+## Vendor ID Security
+
+The devices are not upgradable and thus require no vendor ID set.
+
+## External Interface Access
+
+This plugin requires read only access to `/dev/mei0`.
```

Comment:
presumably it also needs an ioctl to the kernel too?

Reply:
sure

Reply:
Fixed.